src/lib/indexer/__tests__/index.test.ts

```diff
@@ -131,10 +131,12 @@ describe('Indexer', function () {
       ])
       const node = withHappyChainNode()
       const handleBlock = sinon.stub().resolves({})
+      db.getNextUnprocessedBlockAtHeight = sinon.stub().resolves({ hash: '2-hash' })
```

Comment:
Stub should be set in `withInitialLastProcessedBlock`. You can always pass through a parameter of the hash to resolve to.